### PR TITLE
Added Flixius demo (side-scrolling shooter)

### PR DESCRIPTION
**Pull request by SeiferTim**

### CODE REVIEW

File: Tutorials/Flixius/source/PlayState.hx [hunk 1]
```diff
@@ -0,0 +1,644 @@
...
+	
+	public function returnFromSubState():Void
+	{
+		_showingReady = true;
+		#if flash
+		FlxG.sound.playMusic(AssetPaths.music__mp3);
+		#else
+		FlxG.sound.playMusic(AssetPaths.music__ogg);
+		#end
+		/* okay, I admit, this is ridiculous, but it works ;) I would probably find a nicer solution for a 'real' game lol */
```
Comment (Gama11) [inline Tutorials/Flixius/source/PlayState.hx]: Wtf?

File: Tutorials/Flixius/source/Reg.hx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package;
+
+import flixel.util.FlxSave;
+
+/**
+ * Handy, pre-built Registry class that can be used to store 
+ * references to objects and other things for quick-access. Feel
+ * free to simply ignore it or change it in any way you like.
+ */
+class Reg
```
Comment (Gama11) [inline Tutorials/Flixius/source/Reg.hx]: Is `Reg` actually used anywhere?

Comment (SeiferTim) [inline Tutorials/Flixius/source/Reg.hx]: I'm using it for the score... since... it's already there, I guess?

File: Tutorials/Flixius/source/PlayState.hx (hunk 1 above)
Comment (SeiferTim) [inline Tutorials/Flixius/source/PlayState.hx]: Lol I haven't looked at this code in awhile actually, I just verified that it ran...

File: Arcade Classics/Flixius/source/PlayState.hx [hunk 3]
```diff
@@ -0,0 +1,644 @@
...
+	
+	public function returnFromSubState():Void
+	{
+		_showingReady = true;
+		#if flash
+		FlxG.sound.playMusic(AssetPaths.music__mp3);
+		#else
+		FlxG.sound.playMusic(AssetPaths.music__ogg);
+		#end
+		/* okay, I admit, this is ridiculous, but it works ;) I would probably find a nicer solution for a 'real' game lol */
```
Comment (Gama11) [inline Arcade Classics/Flixius/source/PlayState.hx]: I don't feel comfortable having this abomination in a demo. ;) Why is this necessary?

Comment (SeiferTim) [inline Arcade Classics/Flixius/source/PlayState.hx]: This is just the 'Ready' Text, and the box(es) behind it and stuff, it flies in, flashes, flies out and triggers the game to start when it does..

Comment (Gama11) [inline Arcade Classics/Flixius/source/PlayState.hx]: Could you try finding a solution that doesn't require this hack?
